### PR TITLE
Return the last result from GCS

### PR DESCRIPTION
**Pull request by Mark-Hetherington**
The final result includes information about the file.

### CODE REVIEW

File: src/Upload.js [hunk 1]
```diff
@@ -95,6 +95,7 @@ export default class Upload {
       debug(` - End: ${end}`)
 
       const res = await safePut(opts.url, chunk, { headers })
+      this.last_res = res;
```
Comment (oliverwoodings) [inline src/Upload.js]: Casing doesn't match the rest of this project - can you use camel case for this please?

Comment (Mark-Hetherington) [reply src/Upload.js]: Yes, sure thing.

I also figured it's a good idea to initialise the variable to null, rather than undefined.

**Comment by oliverwoodings** [comment]
Thanks for the contribution!

**Comment by Mark-Hetherington** [comment]
Thanks for publishing!

**Pull request merged**